firebase-firestore/src/test/java/com/google/firebase/firestore/local/LocalSerializerTest.java

```diff
@@ -202,18 +202,6 @@ public void testTransformAndTransformThrowError() {
     assertThrows(AssertionError.class, () -> serializer.decodeMutationBatch(batchProto));
   }
 
-  // TODO(b/174608374): Remove these tests once we perform a schema migration.
-  @Test
-  public void testOnlyTransformThrowsError() {
```

Comment:
FWIW You could still assert that the first mutation is not a transform. Optional.

Reply:
Eh, I'll pass on this one since I don't want to back-back-backport it to iOS again.